po/ru.po

```diff
@@ -0,0 +1,355 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Jos Lemmens <jos@jlemmens.nl>
+# This file is distributed under the same license as the ebook-speaker package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ebook-speaker 6.2\n"
+"Report-Msgid-Bugs-To: https://github.com/book-readers/ebook-speaker/issues\n"
+"POT-Creation-Date: 2021-08-21 13:35+0200\n"
+"PO-Revision-Date: 2022-02-15 13:04+0300\n"
+"Last-Translator: Vsevolod Popov <sevapopov13@gmail.com>\n"
+"Language-Team: Vsevolod Popov <sevapopov13@gmail.com>\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: src/alsa_ctl.c:38
+msgid "Cannot read /proc/asound/cards"
+msgstr "Невозможно прочесть /proc/asound/cards"
+
+#: src/common.c:33 src/common.c:294 src/common.c:510 src/ebook-speaker.c:2111
+#: src/ebook-speaker.c:2122
+msgid "Please try to play this book with daisy-player"
+msgstr "Пожалуйста, попробуйте воспроизвести эту книгу с помощью Daisy-плеера"
+
+#: src/common.c:773
+#, c-format
+msgid "Cannot read type: %d"
+msgstr "Невозможно прочесть тип: %d"
+
+#: src/common.c:838
+msgid "Go to page number:"
+msgstr "Перейти на страницу: "
+
+#: src/common.c:856 src/ebook-speaker.c:1674 src/ebook-speaker.c:2627
+#: src/ebook-speaker.c:2932
+msgid "Please wait..."
+msgstr "Пожалуйста, подождите..."
+
+#: src/common.c:1096
+msgid "Select a soundcard:"
+msgstr "Выберите звуковую карту:"
+
+#: src/daisy2.02.c:49 src/daisy2.02.c:130 src/daisy2.02.c:236
+#: src/ebook-speaker.c:2514
+#, c-format
+msgid "Cannot read %s"
+msgstr "Невозможно прочесть %s"
+
+#: src/ebook-speaker.c:304
+msgid "Select a Text-To-Speech application"
+msgstr "Выберите синтезатор речи"
+
+#: src/ebook-speaker.c:322
+msgid "Maximum number of TTS's is reached."
+msgstr "Достигнут максимальный набор допустимых синтезаторов речи"
+
+#: src/ebook-speaker.c:682 src/list_dir.c:55 src/list_dir.c:251
+msgid "'h' for help"
+msgstr "'h' для справки"
+
+#: src/ebook-speaker.c:687
+#, c-format
+msgid "%d pages"
+msgstr "%d страниц"
+
+#: src/ebook-speaker.c:692
+#, c-format
+msgid "level: %d of %d"
+msgstr "Уровень: %d из %d"
+
+#: src/ebook-speaker.c:696
+#, c-format
+msgid "total phrases: %d"
+msgstr "Всего фраз: %d"
+
+#: src/ebook-speaker.c:782 src/list_dir.c:165
+msgid "These commands are available in this version:"
+msgstr "Следующие команды доступны в этой версии:"
+
+#: src/ebook-speaker.c:786 src/list_dir.c:169
+msgid "cursor down,2     - move cursor to the next item"
+msgstr "курсор вниз,2    - переместить курсор на следующий элемент"
```

Comment:
This is still not aligned with the other Russian translations :)

Reply:
That's strange, I make it so all dashes are in the column 26, including this line. Are you talking about the down arrow line?

Reply:
Oh, sorry it's actually the english piece which is unaligned, the russian one is fine

Reply:
It seems what happened is that you modified the english version

Reply:
you need to set it back to the original version :) perhaps use `diff` against the pot file, to check that the english versions are exactly as in the pot file

Reply:
Oh, yes. That's true. It happened accedentaly. I fixed it and will publish changes now. Do I understand correctly that msgid are lines that will be mapped with msgstr translations so id is like mapping?

Reply:
msgid are a sort of index yes. If they don't exactly match what was in the .pot file, gettext will not be able to find the mapped translation, that's why it's important that they are kept unmodified